src/main/resources/application.conf

```diff
@@ -36,6 +36,12 @@ workflow-options {
 
   // AES-256 key to use to encrypt the values in `encrypted-fields`
   base64-encryption-key: "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="
+
+  // Directory where to write per workflow logs
+  workflow-log-dir: "cromwell-workflow-logs"
+
+  // When true, per workflow logs will be deleted after copying
+  workflow-log-temporary: true
```

Comment:
hmm is `true` a good default value for this?

I tried running a workflow through a vanilla Cromwell and it created the `cromwell-workflow-logs` and indeed did remove the logs after the workflow finished... but it seems to me like the default behavior should be to keep workflow logs and set this option to true for the Firecloud case